nodejs/week1/homework/src/backend/app.js

```diff
@@ -61,11 +60,13 @@ app.get("/reservations", async (request, response) => {
 // random reservation
 
 app.get("/reservation", async (request, response) => {
-  const random = Math.round(Math.random() * reservations.length);
-  const reservationRandom = reservations[random]
+  const reservationRandom = reservations[randomProject(reservations)]
   response.json(reservationRandom);
 });
 
+function randomProject(a){
```

Comment:
Tip:
By passing in an array here, you limit this function to only be used with arrays.
If you instead passed in the length you could use this function in any scenario where you have a number.
`randomProject(reservations.length)`

Reply:
Oh, yes. You are totally right.  Thanks a lot. :)